packages/react-scripts/template/src/utils/is-promise.js

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
This file is not used anywhere. You can probably leave it out. 
same for `is-promise.test.js`